man/sd_bus_call_method.xml

```diff
@@ -96,14 +96,24 @@
     <citerefentry><refentrytitle>sd_bus_message_new_method_call</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
     <citerefentry><refentrytitle>sd_bus_message_append</refentrytitle><manvolnum>3</manvolnum></citerefentry> and
     <citerefentry><refentrytitle>sd_bus_call</refentrytitle><manvolnum>3</manvolnum></citerefentry>
-    functions into a single function call.</para>
+    functions into a single function call. When <parameter>reply</parameter> message returned from this function
+    is no longer needed, it should be unreferenced by calling
+    <citerefentry><refentrytitle>sd_bus_message_unref</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    on it.</para>
 
     <para><function>sd_bus_call_method_async()</function> is a convenience function for initializing
     a bus message object and calling the corresponding D-Bus method asynchronously. It combines the
     <citerefentry><refentrytitle>sd_bus_message_new_method_call</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
     <citerefentry><refentrytitle>sd_bus_message_append</refentrytitle><manvolnum>3</manvolnum></citerefentry> and
     <citerefentry><refentrytitle>sd_bus_call_async</refentrytitle><manvolnum>3</manvolnum></citerefentry>
     functions into a single function call.</para>
+
+    <para>The message <parameter>m</parameter> passed to the callback is only borrowed, that is, the callback should
+    not call <citerefentry><refentrytitle>sd_bus_message_unref</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    on it. If the callback wants to hold on to the message beyond the lifetime of the callback, it needs to call
+    <citerefentry><refentrytitle>sd_bus_message_ref</refentrytitle><manvolnum>3</manvolnum></citerefentry> to create a
+    new reference.</para>
```

Comment:
Hm, this is copied from sd_bus_call(3). I do not think we should duplicate such...

Reply:
actually it's an important point, so either the same wording or a reference to the exact paragraph is a good thing to have here too, better to have too much info than too little

Reply:
Then, let's extract this to another .xml file and include it for all sd_bus_*_async functions.